app/src/main/java/woowacourse/movie/model/PriceCalculator.kt

```diff
@@ -0,0 +1,22 @@
+package woowacourse.movie.model
+
+import woowacourse.movie.model.policy.MorningPolicy
+import woowacourse.movie.model.policy.MovieDayPolicy
+import woowacourse.movie.model.policy.NightPolicy
+import java.time.LocalDate
+import java.time.LocalTime
+
+object PriceCalculator {
+    private val policies = listOf(MovieDayPolicy(), MorningPolicy(), NightPolicy())
+    fun calculate(
+        price: Price,
+        playingDate: LocalDate,
+        playingTime: LocalTime
+    ): Price {
+        var calculatePrice = price
+        for (policy in policies) {
+            calculatePrice = policy.calculate(playingDate, playingTime, calculatePrice)
+        }
+        return calculatePrice
```

Comment:
```suggestion
        return policies.fold(price) { total, item -> item.calculate(playingDate, playingTime, total) }
```
이렇게 작성해보실래요? var과 iterator를 사용하는 다른 곳에서 fold, reduce, map, filter 등 코틀린 연산자를 활용할 수 있는 방법들도 고민해보세요.

Reply:
제가 추가 커밋을 푸쉬를 안하고 리뷰 요청을 했었네요!! ㅠㅠ 이렇게 수정했습니다!